### ISSUE TITLE
Using the Articles for Creation help desk template

### ISSUE THREAD

**Issue by enterprisey** [azatoth/twinkle#220]
The [AFCHD/u](//en.wikipedia.org/wiki/Template:AFCHD/u) template should be used instead of the Talkback template in the Talkback module when "Wikipedia:Articles for Creation/Help desk" is typed in as the full page name.
